support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala

```diff
@@ -9,9 +9,9 @@ import org.joda.time.DateTime
 import scala.concurrent.{ExecutionContext, Future}
 
 class ContributionEmailFields(
-  getMandate: String => Future[Option[String]],
-  created: DateTime,
-) {
+                               getMandate: String => Future[Option[String]],
+                               created: DateTime,
+                             ) {
```

Comment:
I guess we don't want this?

Reply:
Good shout. I just left this alone, but yeah, it's only being used in unit tests now. @tomrf1 are you happy for this to be removed?

Reply:
Oh I just meant the formatting!

Reply:
would definitely advocate for removing unused code though!

Reply:
I was just about to comment again saying that I missed another use of the type! I'm not firing on all cylinders this week! Yeah, I'll fix the inconsistent formatting.

Reply:
looks like there's still an extra space there!